sdk/formrecognizer/Azure.AI.FormRecognizer/tests/FormRecognizerClient/FormRecognizerClientLiveTests.cs

```diff
@@ -1918,6 +1918,174 @@ public async Task StartRecognizeInvoicesWithMultiplePageArgument(string page1, s
+            Assert.True(form.Fields.ContainsKey("LastName"));
+            Assert.True(form.Fields.ContainsKey("Region"));
+            Assert.True(form.Fields.ContainsKey("Sex"));
+
+            Assert.AreEqual("123 STREET ADDRESS YOUR CITY WA 99999-1234", form.Fields["Address"].Value.AsString());
+            Assert.AreEqual("LICWDLACD5DG", form.Fields["DocumentNumber"].Value.AsString());
+            Assert.AreEqual("LIAM R.", form.Fields["FirstName"].Value.AsString());
+            Assert.AreEqual("TALBOT", form.Fields["LastName"].Value.AsString());
+            Assert.AreEqual("Washington", form.Fields["Region"].Value.AsString());
+
+            Assert.That(form.Fields["Country"].Value.AsCountryCode(), Is.EqualTo("USA"));
+            Assert.That(form.Fields["Sex"].Value.AsGender(), Is.EqualTo(FieldValueGender.M));
+
+            var dateOfBirth = form.Fields["DateOfBirth"].Value.AsDate();
+            Assert.AreEqual(6, dateOfBirth.Day);
+            Assert.AreEqual(1, dateOfBirth.Month);
+            Assert.AreEqual(1958, dateOfBirth.Year);
+
+            var dateOfExpiration = form.Fields["DateOfExpiration"].Value.AsDate();
+            Assert.AreEqual(12, dateOfExpiration.Day);
+            Assert.AreEqual(8, dateOfExpiration.Month);
+            Assert.AreEqual(2020, dateOfExpiration.Year);
+        }
+
+        [RecordedTest]
+        public async Task StartRecognizeIdDocumentsIncludeFieldElements()
+        {
+            var client = CreateFormRecognizerClient();
+            var options = new RecognizeIdDocumentsOptions() { IncludeFieldElements = true };
+            RecognizeIdDocumentsOperation operation;
+
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.DriverLicenseJpg);
+            using (Recording.DisableRequestBodyRecording())
+            {
+                operation = await client.StartRecognizeIdDocumentsAsync(stream, options);
+            }
+
+            RecognizedFormCollection recognizedForms = await operation.WaitForCompletionAsync(PollingInterval);
+
+            var form = recognizedForms.Single();
+
+            ValidatePrebuiltForm(
+                form,
+                includeFieldElements: true,
+                expectedFirstPageNumber: 1,
+                expectedLastPageNumber: 1);
+        }
+
+        [RecordedTest]
+        public async Task StartRecognizeIdDocumentsCanParseBlankPage()
+        {
+            var client = CreateFormRecognizerClient();
+            RecognizeIdDocumentsOperation operation;
+
+            using var stream = FormRecognizerTestEnvironment.CreateStream(TestFile.Blank);
+            using (Recording.DisableRequestBodyRecording())
+            {
+                operation = await client.StartRecognizeIdDocumentsAsync(stream);
+            }
+
+            RecognizedFormCollection recognizedForms = await operation.WaitForCompletionAsync(PollingInterval);
+
+            Assert.IsEmpty(recognizedForms);
```

Comment:
This new endpoint behaves differently with empty pages. If it can't locate ID document fields, it will simply not return the document result instead of returning a document result with no fields. Service is already aware of the behavior. Since it's in accordance with the swagger, there's no intent to change it.

This makes testing a bit harder for multipage files. If we pass a multipage invoice, for instance, it will not detect the right fields and won't return any recognized forms, which makes it impossible for us to make multipage tests for this endpoint. I have opened an issue to track it (https://github.com/Azure/azure-sdk-for-net/issues/20019) until we get better testing files.